packages/sanity/src/structure/components/requestPermissionDialog/RequestPermissionDialog.tsx

```diff
@@ -169,11 +168,10 @@ export function RequestPermissionDialog({
                   value={note}
                   onChange={(e) => {
                     setNote(e.currentTarget.value)
-                    setNoteLength(e.currentTarget.value.length)
                   }}
                 />
 
-                <Text align="right" muted size={1}>{`${noteLength}/${MAX_NOTE_LENGTH}`}</Text>
+                <Text align="right" muted size={1}>{`${note.length}/${MAX_NOTE_LENGTH}`}</Text>
```

Comment:
One refactor change to derive the note length from the `note` state instead of having a separate state value for it - again removing copy pasta from another file